runtime/doc/windows.txt

```diff
@@ -263,28 +263,54 @@ left of the Vim window.
 Closing a window
 ----------------
 
+:q[uit]
+:{count}q[uit]
```

Comment:
same regarding brackets (there's more, but I don't want to flood your inbox). If this is inconsistent throughout, perhaps a separate PR should deal wtih it.